service/controller/v12/templates/cloudformation/guest/instance.go

```diff
@@ -4,6 +4,9 @@ const Instance = `{{define "instance"}}
   {{ .Instance.Master.Instance.ResourceName }}:
     Type: "AWS::EC2::Instance"
     Description: Master instance
+    DependsOn:
```

Comment:
Problem was that VM did not depend on volumes to it was immediately created with etcd as first disk and in that time docker disk was resized (takes 3-5 minutes) and then docker disk attached as second disk and mounted to `/var/lib/etcd`.

This dependency makes sure that VM will wait to both volumes to be ready before get started.

Reply:
Very good catch, thanks!